docs/packaging.md

```diff
@@ -250,37 +250,59 @@ apt install aziot-identity-service
 aziot init
 ```
 
-The user installs the package, then runs `aziot init` to set up the configuration with minimal information like the device provisioning method.
+After installng the package, run `aziot init` to interactively set up the configuration with minimal information like the device provisioning method.
+
 
+## Installation procedure for IoT Edge (`aziot-edge`)
+
+```sh
+apt install aziot-edge
+
+iotedge init
+```
 
-## Installation procedure (`iotedge-aziot`)
+After installing the `aziot-edge` package, run `iotedge init` to interactively set up the configuration. It performs the initialization for both the IS+KS+CS components installed by the `aziot-identity-service` package and the MR component installed by the `aziot-edge` package.
 
+## Updating from `iotedge` to `aziot-edge`
 ```sh
-apt install iotedge-aziot
+apt remove iotedge libiothsm-std 
+
+apt install aziot-edge
 
 iotedge init
 ```
 
-The user installs the package, then runs `iotedge init` to set up the configuration of the IS+KS+CS+MR components.
+You must remove the existing `iotedge` and `libiothsm-std` packages before installing the `aziot-edge` package (or even the `aziot-identity-service` package). We do not want a situation where the services from both packages are running at the same time. They would step over each other trying to provision the device and manage Docker modules. We enforce mutual exclusivity between the packages by having them conflict with each other so that the distribution's package manager does not allow them both to be installed at the same time.
 
+The `iotedge init` automatically detects when the configuration of the old IoT Edge installation is available and prompts whether to create the new configuration based on the old. Agreeing to import the old configuation does not remove it. This allows downgrading from `aziot-edge` to `iotedge`.
```

Comment:
What do we mean by "this allows downgrading from ... to .."? The old configuration file shouldn't be overwritten in place, right? 

I am not sure if this line is necessary. Or we can share the details in the downgrade step.

Reply:
I'll remove the last sentence as it isn't necessary here.

Reply:
Yeah, I saw the downgrade step has details laid out already. Here having the line could be confusing.